octoprint_mrbeam/__init__.py

```diff
@@ -36,6 +36,7 @@
 
 from octoprint_mrbeam.rest_handler.update_handler import UpdateRestHandlerMixin
 from octoprint_mrbeam.util.connectivity_checker import ConnectivityChecker
+from .iobeam.airfilter import airfilter
```

Comment:
Can we change this from relative to absolute import?

Reply:
sure